CONTRIBUTING.md

```diff
@@ -135,4 +135,10 @@ To make this easier, we do appreciate it if folks enable the "Allow edits from m
 
 ## On ideologically motivated changes
 
-This is a purely technical project. As such, it is not an appropriate arena to advertise your personal politics or religious beliefs. Any changes that appear ideologically motivated will be rejected.
+Serenity's goal is to enable collaboration between as many groups as *reasonably* possible, and we welcome contributions that make the project more accessisble to people.
+
+However, Serenity is intended to be a purely technical exercise, in the sense of it not being meant to invoke any sociopolitical change. We explicitly try to avoid drama or getting involved in "external" culture wars, and can reject changes that we feel to be sensitive.
```

Comment:
This change suggestion aligns with previous feedback - I'd say that technical writing or design work or translating technical terms is also "technical work" - otherwise I'd have used the word "engineering" or "engineering exercise", but it seems like this is just me.

Reply:
I understand where you're coming from. My perspective is that working on such a big project involves collaboration, which is a social thing rather than technical. Contributors come here to (in no particular order):
- have fun
- collaborate
- build something cool

The phrase "purely technical" may come across as dismissive of the human aspects, possibly resulting in a sterile and corporate-like image. I hope this isn't the intent here. :P

Reply:
It's not, and I fully get where *you're* coming from - just tried to give background as to why I wrote this to begin with in the first place.